client/main.cpp

```diff
@@ -176,7 +176,7 @@ static int create_native(char **args)
         is_clang = true;
     // Args[0] may be a compiler or the first extra file.
     if (args[0] && ((!strcmp(args[0], "clang") && (is_clang = true))
-                    || (!strcmp(args[0], "gcc") && (is_clang = false)))) {
+                    || (!strcmp(args[0], "gcc") && !(is_clang = false)))) {
```

Comment:
If this is merged you may close https://github.com/icecc/icecream/pull/124 since it contains essentially this fix. Your way of fixing it is terser and probably readable enough for the audience anyway.